op_crates/fetch/21_file.js

```diff
@@ -287,8 +287,57 @@
     }
   }
 
-  window.__bootstrap.blob = {
+  const _name = Symbol("[[Name]]");
+  const _lastModfied = Symbol("[[LastModified]]");
+
+  class File extends Blob {
+    /** @type {string} */
+    [_name];
+    /** @type {number} */
+    [_lastModfied];
+
+    /**
+     * @param {BlobPart[]} fileBits 
+     * @param {string} fileName 
+     * @param {FilePropertyBag} [options] 
+     */
+    constructor(fileBits, fileName, options) {
```

Comment:
```suggestion
    constructor(fileBits, fileName, options = {}) {
```

Would obviate the need for the optional chaining below and should still be spec compliant.

Reply:
Not really. When passing `null` as the third parameter, the `= {}` wouldn't catch, and thus options could still not be a WebIDL dictionary. The optional chaining prevents this.